### PR TITLE
Use base Java Docker image on Dockerhub

### PR DESCRIPTION
**Pull request by dwaynebailey**
https://tools.hmcts.net/jira/browse/RDM-2687

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: DockerHubfile [hunk 1]
```diff
@@ -17,16 +17,17 @@ RUN mkdir -p application/src/main/resources/META-INF \
 RUN gradle installDist
 
 #### Actual DockerFile
-FROM openjdk:8-jre
```
Comment (vlaurin) [inline DockerHubfile]: The name of this fine is inconsistent with the other `hub.Dockerfile` no?

Comment (dwaynebailey) [reply DockerHubfile]: Good catch, thanks

Comment (dwaynebailey) [reply DockerHubfile]: You got the `- FROM openjdk:8-jre` but missed the `+ FROM hmcts/cnp-java-base:openjdk-jre-8-alpine-1.4`

Comment (dwaynebailey) [reply DockerHubfile]: Doh read the comment @dwaynebailey

**Pull request merged**